docs/outputs.rst

```diff
@@ -150,4 +150,66 @@ I012   ignored            ign_add1
 
 Visual reports
 --------------
-We're working on it.
+Static visual reports can be generated by using the ``--png`` flag when calling 
+tedana from the command line. Images are created and placed within the output 
+directory, in a folder labeled ``figures``. 
+
+These reports consist of three main types of images.
+
+Component Images
+````````````
+.. image:: /_static/example_good_component.png
+  :align: center
+  
+For each component identified by tedana, a single image will be created. Above is an
+example of an accepted component. These are designed for an up-close inspection of both
+the spatial and temporal aspects of the component, as well as ancilliary information. 
+
+The title of the plot provides information about variance, kappa and rho values
+as well as the reasons for rejection, if any (see above for codes). 
+
+Below this is the component timeseries, color coded on the basis of its
+classification. Green for accepted, Red for rejected, Black for ignored or
+unclassified. 
+
+Slices are then selected from sagittal, axial and coronal planes, to highlight the
+component pattern. By default these images used the red-blue colormap and are scaled
+to 10% of the max beta value. 
+
+Note: You can select your own colormap to use by specifying its name when calling tedana
```

Comment:
Can you make this a note box? Just use `..note:` before and tab out everything in the note.

Reply:
Made the note, a ..note:, and I believe I have fixed sentences, though I am not sure of the purpose of doing that? I'm new to rst (note that I just copied the figure reference bits), so any other feedback is appreciated.